sdk-lib/src/main/java/cash/z/ecc/android/sdk/internal/transaction/PagedTransactionRepository.kt

```diff
@@ -105,12 +106,16 @@ internal class PagedTransactionRepository private constructor(
         }
     }
 
-    // TODO: begin converting these into Data Access API. For now, just collect the desired operations and iterate/refactor, later
+    // TODO [#681]: begin converting these into Data Access API. For now, just collect the desired
+    //  operations and iterate/refactor, later
+    // TODO [#681]: https://github.com/zcash/zcash-android-wallet-sdk/issues/681
     suspend fun findBlockHash(height: BlockHeight): ByteArray? = blocks.findHashByHeight(height.value)
     suspend fun getTransactionCount(): Int = transactions.count()
 
-    // TODO: convert this into a wallet repository rather than "transaction repository"
+    // TODO [#681]: convert this into a wallet repository rather than "transaction repository"
+    // TODO [#681]: https://github.com/zcash/zcash-android-wallet-sdk/issues/681
 
+    @Suppress("LongParameterList")
```

Comment:
Let's move this to the `new` method instead of being on the companion object, in order to narrow the scope of this suppression.